backend/.gitignore

```diff
@@ -0,0 +1,33 @@
+HELP.md
```

Comment:
Why two gitignore files?? One should be enough

Reply:
You are right, one should be removed, the thing is that I have a another repo with some setup for my office pc, so let me just check what is the one that I should keep from this repo, probably the one from backend folder will be the one that I will remove.